app/(guarded)/songs/AlbumTile.jsx

```diff
@@ -0,0 +1,17 @@
+import AlbumCover from '../now-playing/AlbumCover';
+
+export default function AlbumTile({
+  track: { albumCoverUrl, trackName, artist }
+}) {
+  return (
+    <div className="flex flex-col items-center font-primary p-8">
+      <AlbumCover classNames="h-[12rem] w-[12rem]" src={albumCoverUrl} />
+      <p className="font-bold text-2xl text-primary pt-2 pb-1 text-center">
```

Comment:
```suggestion
      <p className="font-bold text-2xl text-primary pt-2 pb-1 text-center line-clamp-2">
```
Use line clamp to limit lines so long names don't make the rows too tall and the grid looks more consistent. 
Install and configure tailwind with this package: https://github.com/tailwindlabs/tailwindcss-line-clamp